src/components/BetForm/BetForm.tsx

```diff
@@ -43,37 +44,32 @@ export const BetForm = ({
   return (
     <div
       className={
-        gameStatus === "finished" || gameStatus === "notstarted"
-          ? "block-start-hidden "
-          : "block-start"
+        gameStatus == GameStatus.finished || gameStatus == GameStatus.notstarted
+          ? styles.block__hidden
+          : styles.block__visible
       }
     >
-      {gameStatus === "start" ? (
-        <div className="bet">
-          <form onSubmit={handleSubmit(onSubmit)} className="bet__form">
-            <label className="bet__label">enter your bet</label>
+      {gameStatus == GameStatus.start ? (
```

Comment:
== ?

Reply:
i tried to do like that - it didn't work